scripts/compile.ts

```diff
@@ -33,10 +33,13 @@ async function writeFactoryDeployerTransaction(contract: CompilerOutputContract,
 	const data = arrayFromHexString(deploymentBytecode)
 
 	if (!process.env.MNEMONIC) throw Error("MNEMONIC is required")
-	const signer = ethers.Wallet.fromMnemonic(process.env.MNEMONIC!!)
-	const signedEncodedTransaction = await signer.signTransaction({
-		nonce, gasPrice, gasLimit, value, data, chainId
-	})
+    const provider = new ethers.providers.JsonRpcProvider(process.env.RPC)
+    const signer = ethers.Wallet.fromMnemonic(process.env.MNEMONIC!!).connect(provider)
+    //if the network supports EIP-1155 transaction, use the EIP-1155 transaction type
```

Comment:
the comment seems to mention the incorrect eip number

Reply:
thank you for your review, it's fixed now.